### ISSUE TITLE
Accuracy

### ISSUE THREAD

**Issue by HassamChundrigar** [Wrymm/Random-Survival-Forests#1]
How to Calculate the Accuracy?

**Comment by Wrymm** [comment]
Hi,
The output of this model is a probability, hence you have to choose a threshold to make your decision. This way, it is not common practice to compute an accuracy. We rather usually rely on AUC, please have a look at [this thread](https://developers.google.com/machine-learning/crash-course/classification/roc-and-auc) for more details. You can find tools to compute it in your favourite ML library (such as [sklearn](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.auc.html)).

**Comment by salvationarinze** [comment]
Hi,
How will I convert the predicted probability to a value that could be evaluated with c-index. 
Thanks
